fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -118,6 +120,13 @@
     // only for persistence param
     private boolean isJobTypeRead = false;
 
+    // number of rows processed on BE, this number will be updated periodically by query report.
+    // A load job may has several load tasks, so the map key is load task's plan load id.
+    protected Map<TUniqueId, AtomicLong> numLoadedRows = Maps.newConcurrentMap();
```

Comment:
I think it's better to make RuntimeStatics, not only numLoadedRows.
And it name should be xxxMap.

Reply:
Changed